src/components/Profile/ProfileInfo/ProfileInfo.js

```diff
@@ -1,24 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 
-import ProfileStatusWithHooks from './ProfileStatusWithHooks';
+import ProfileStatus from './ProfileStatus';
+import ProfileData from './ProfileData/ProfileData';
+import ProfileDataForm from './ProfileData/ProfileDataForm';
 
 import Loader from '../../../utils/loader';
+import { AVATAR_DEFAULT } from '../../../utils/url-utils';
 
 import style from './ProfileInfo.module.css';
 
 const ProfileInfo = (props) => {
-  const { profile, status, updateStatus } = props;
+  const [editMode, setEditMode] = useState(false);
+
+  const {
+    profile, status, updateStatus, isOwner, savePhoto, saveProfile,
```

Comment:
The number of things on a line is more than 3 - keep such things on a new line.

Reply:
/fxd